.pre-commit-config.yaml

```diff
@@ -116,6 +116,14 @@ repos:
       files: .*gitlab.*\.yml$
       pass_filenames: false
       stages: [pre-push]
+    - id: gitlab-lint-jobs-codeowners
+      name: gitlab-lint-jobs-codeowners
+      description: lint the gitlab configuration to verify jobs codeowners
+      entry: 'inv linter.gitlab-ci-jobs-codeowners'
+      language: system
+      require_serial: true
+      files: .*gitlab.*\.yml$
+      pass_filenames: false
```

Comment:
Should it be a `pre-push` (I don't know the duration of the linter)

Reply:
This linter is quite fast so a pre-commit is ok (only modified files are checked). You can try `inv linter.gitlab-ci-jobs-codeowners --all-files` to lint all the files and it takes 0.25s

Reply:
(but the other linters that are not added in this PR are much slower)